PROTOCOL.md

```diff
@@ -98,6 +98,10 @@ displayed inline because they happen too often. For instance, the main server
 gets around 5 joins/leaves a second, and showing that inline with chat would
 make it near-impossible to chat.
 
+Another message type is `m` followed by a timestamp from the server, followed
+by the message.  Example: `|m|timestamp|message`, this will make it so that the
+message will have a timestamp, if a user has those enabled for their settings.
+
```

Comment:
Put it next to `|c|` but I'm still thinking about it...

Also `|m|timestamp|message` is completely wrong; it's `|m:|timestamp|message` or `|m|message`.